workshop/main.rb

```diff
@@ -0,0 +1,50 @@
+require_relative "./selenium.rb"
+class LoginFlow
+    extend SeleniumFramework
+
+    def self.login
+        setup("https://www.cocoleni.in/")
```

Comment:
You shouldn't setup for every automation module. You only need to do it once. Refactor the code to another method, like `navigate_to_website`. 
You can pass the driver object to the setup only ONCE

Reply:
Done